### ISSUE TITLE
paper #542:  Robotics Software Engineering:

### ISSUE THREAD

**Issue by kstolee** [researchart/fse20#120]
This issue is for discussion related to PR #8 

Author contact: @SergioGarG

**Comment by kstolee** [comment]
# Authors list
## Author 1 (corresponding author)
Sergio García
Email: sergio.garcia@gu.se
Github id: SergioGarG

## Author 2
Daniel Strüber 
Email: d.strueber@cs.ru.nl
Github id: dstrueber

## Author 3
Davide Brugali
Email: davide.brugali@unibg.it
Github id: brugali

## Author 4
Thorsten Berger
Email: thorsten.berger@chalmers.se
Github id: thorstenberger

## Author 5
Patrizio Pelliccione
Email: patrizio.pelliccione@cse.gu.se
Github id: PatrizioPelliccione

**Comment by art-reviewer** [comment]
[Paper](https://github.com/SergioGarG/fse20/blob/a0d0082e94474b534b2beaec296d09a780dc7e38/submissions/functional/fse-542/esecfse2020-paper542.pdf)

**Comment by itsreallyhotoutside** [comment]
@gousiosg and @kstolee this paper's artifact is coded interviews and survey results with an R script to analyze the survey results and is applying for the functional badge as opposed to available badge. Does this fulfill the criteria for functionality?

**Comment by gousiosg** [comment]
Great question. As the short description of the functional badge mentions: "Available + Artifacts documented, consistent, complete, exercisable, and include appropriate evidence of verification and validation". If they satisfy those properties as analysed in the [artifact reviewing guide](https://www.acm.org/publications/policies/artifact-review-badging) I would say yes. 

@kstolee ?

**Comment by gousiosg** [comment]
@a-hopefully-friendly-reviewer We would appreciate your review till Monday morning AoE.

**Comment by kstolee** [comment]
> Great question. As the short description of the functional badge mentions: "Available + Artifacts documented, consistent, complete, exercisable, and include appropriate evidence of verification and validation". If they satisfy those properties as analysed in the [artifact reviewing guide](https://www.acm.org/publications/policies/artifact-review-badging) I would say yes.
> 
> @kstolee ?

Agreed. If they meet the documented + consistent + complete + exercisable, then functional would be appropriate.

**Comment by hopefully-anon** [comment]
I find the artifact for paper 542 to be **functional**

Also I applaud the authors for including the original responses in the artifact, as this data contains many details that might be useful for future analysis.

1. **Installation** :  This criteria does not apply because the artifact does not require code installation.
2. **Public Availability** : ✅ confirmed. 
3. **Documentation** : ✅ Identifies and describes the contents of the artifact and how to use it to generate the paper's results.
4. **Consistent/Complete** : ✅ The graphs generated by the `r` script are those presented in the paper.  Further, the authors include the original responses to the data and the results of the open coding process.
5. **Exercisable** : ✅ the `analysis-script.r` script runs as described and is documented.

**Comment by itsreallyhotoutside** [comment]
@SergioGarG thanks for the submission!

- [x] Available: The artifact is placed on a publicly accessible archival repository.

- [x] Functional : 
    - [x] Documented: At minimum, an inventory of artifacts is included, and sufficient description provided to enable the artifacts to be exercised.

    - [x] Consistent: The artifacts are relevant to the associated paper, and contribute in some inherent way to the generation of its main results.

    - [x] Complete: To the extent possible, all components relevant to the paper in question are included.

    - [x] Exercisable:  Included scripts and/or software used to generate the results in the associated paper can be successfully executed, and included data can be accessed and appropriately manipulated.

Notes:

- The R script was exercisable and documented. All the data that is relevant to the paper is present in the artifact which is helpful for a replication. Additionally, the data is publicly hosted and has a DOI.

**Comment by kstolee** [comment]
Thanks @itsreallyhotoutside and @hopefully-anon for the reviews. It seems you are in agreement that this badge meets the ```available``` and ```functional``` badges. I'll add the labels. Feel free to keep the conversation going as needed.

**Comment by SergioGarG** [comment]
Hi,

Do we need to do something else regarding the badges for the camera-ready version of the paper?

**Comment by SergioGarG** [comment]
> Hi,
> 
> Do we need to do something else regarding the badges for the camera-ready version of the paper?

@kstolee?

**Comment by kstolee** [comment]
@SergioGarG make sure you cite your artifact by putting the DOI in the references section. From there, the publisher takes care of actually placing the cute little badge circles.
